src/components/tasks/card/SavedCheckmark.tsx

```diff
@@ -0,0 +1,16 @@
+import Image from 'next/image';
+import classNames from '@/components/tasks/card/card.module.scss';
+export const SavedCheckmark = () => (
+    <div
+        className={`${classNames.checkmark} ${classNames.selfAlignEnd}`}
+        data-testid="checkmark"
+    >
+        <Image
```

Comment:
can we have smaller 'alt' text like "not found" and onerror attribute as well

Reply:
updated alt text. For onError I need to check what should be the fallback image for this.